src/plugins/image/utils.js

```diff
@@ -1,16 +1,3 @@
-import { SOURCE_TYPE } from './constants.js';
```

Comment:
I deleted the second part of this module (in the already merged PR), so this file can be deleted now.